appveyor.yml

```diff
@@ -5,19 +5,12 @@ environment:
   ruby_version: 24-x64
   TargetPath: sassc/bin/sassc.exe
   matrix:
-  - Compiler: msvc
```

Comment:
Why have these been deleted?

Reply:
See comment above, trying something in Appveyor CI